app/presenters/html_publication_presenter.rb

```diff
@@ -20,7 +20,11 @@ def copyright_year
   end
 
   def format_sub_type
```

Comment:
Unrelated to this PR – this method would be better called "parent_document_type"